postman/schemas/transfer.yml

```diff
@@ -2216,6 +2216,48 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/response401"
+  "/transfers/progress/{quoteHistoryId}":
+    get:
+      tags:
+      - Transfers
+      summary: Get Transfer Progress by Quote History ID
+      description: This operation retrieves transfer status for the transfer associated with a provided quote history ID.
+      operationId: getTransferStatusOfQuote
+      parameters:
+      - name: quoteHistoryId
+        in: path
+        description: The quote history identifier
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/transferProgressResponse"
+              example:
+                transferRecordStatus: "Created"
```

Comment:
This property doesn't match the property on the schema object. This one is transferRecordStatus. The schema shows a property named "status". Which one is correct?

Reply:
transferRecordStatus is right. Fixed the schema